Dockerfile

```diff
@@ -0,0 +1,57 @@
+FROM amazonlinux:2017.03
+
+MAINTAINER akrug@mozilla.com
+
+# Patch this image
+
+RUN yum update -y
+
+# Install python 3.6.1
+
+RUN yum install \
+    gcc wget findutils \
+    zlib zlib-devel openssl-devel \
+    libffi-devel git \
+    -y
+
+WORKDIR /usr/src/
+
+RUN wget https://www.python.org/ftp/python/3.6.1/Python-3.6.1.tgz
+
+RUN tar xzf Python-3.6.1.tgz
+
+WORKDIR /usr/src/Python-3.6.1
+
+RUN ./configure --enable-optimizations
+
+RUN make install
+
+RUN rm /usr/src/Python-3.6.1.tgz
+
+# Install pip
+
+WORKDIR /root
+
+RUN wget https://bootstrap.pypa.io/get-pip.py
```

Comment:
this would also solves the wget|sudo bashes
/me rants

Reply:
Will kill this when Amzn Linux has a package.